Makefile

```diff
@@ -23,7 +23,7 @@ lint: fmt vet
 
 # Run tests
 test: tools generate fmt vet manifests
-	ginkgo -r --randomizeAllSpecs --randomizeSuites --failOnPending --cover -coverprofile=../coverage.out --trace --race --progress
+	ginkgo -r --failOnPending --cover -coverprofile=../coverage.out --trace --race --progress
```

Comment:
Remove the randomize options at least until we address https://github.com/Skyscanner/applicationset-progressive-sync/issues/111
Until the controller isn't scoped to a single namespace, the ginkgo random options might cause different objects to pollute the test namespace.